### PR TITLE
Add TS_SAMPLE_LIST constant

### PR DESCRIPTION
**Pull request by dmytro-ndp**
### What does this PR do?
It adds an ability to define own list of samples to run the test RecomendedExtentions against using `TS_SAMPLE_LIST` parameter.

### Screenshot/screencast of this PR
- RecomendedExtentions test run with default parameters https://main-jenkins-csb-crwqe.apps.ocp-c1.prod.psi.redhat.com/job/Testing/job/e2e/job/basic/job/typescript-tests/7387/console
- RecomendedExtentions test run with `-e TS_SAMPLE_LIST='Python,Quarkus,.NET'`: https://main-jenkins-csb-crwqe.apps.ocp-c1.prod.psi.redhat.com/job/Testing/job/e2e/job/basic/job/typescript-tests/7386/console

### What issues does this PR fix or reference?
It's followup of https://github.com/eclipse/che/pull/22144

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [ ] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix or new feature ](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix-or-new-feature)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Review by ScrewTSW — APPROVED** [review]
LGTM

File: tests/e2e/specs/dashboard-samples/RecomendedExtentions.spec.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ const loginTests: LoginTests = e2eContainer.get(CLASSES.LoginTests);
 const driverHelper: DriverHelper = e2eContainer.get(CLASSES.DriverHelper);
 
 const webCheCodeLocators: Locators = new CheCodeLocatorLoader().webCheCodeLocators;
-const samples: any = ['Node.js MongoDB', 'Node.js Express'];
+const samples: any = TestConstants.TS_SAMPLE_LIST.split(',');
```
Comment (nallikaea) [inline tests/e2e/specs/dashboard-samples/RecomendedExtentions.spec.ts:44]: string[]

Comment (nallikaea) [reply tests/e2e/specs/dashboard-samples/RecomendedExtentions.spec.ts:44]: my mistake, sorry =)

Comment (dmytro-ndp) [reply tests/e2e/specs/dashboard-samples/RecomendedExtentions.spec.ts:44]: don't worry

Thank you for review!

**Pull request merged**